### ISSUE TITLE
the loss is aroung 20 for a long time

### ISSUE THREAD

**Issue by jamiechoi1995** [HiKapok/SSD.TensorFlow#96]
I start from the pretrained VGG-16 model,
but my loss is around 20 no matter how I adjust the learning rate or batchsize

**Comment by HiKapok** [comment]
@jamiechoi1995 please make sure you are following readme.md
